docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -18,3 +20,14 @@ Deploy the Teleport Kubernetes Service, Application Service, or Database Service
 
 </Tile>
 </TileSet>
+
+# Plugins
+
+<TileSet>
+<Tile href="./reference/teleport-plugin-jira.mdx" icon="kubernetes" title="teleport-plugin-jira">
+![Kubernetes agent](../../../img/k8s/mini-diagrams/k8s-to-teleport-mono.svg)
+
+Send notifications about acccess requests and can automatically approve them via PagerDuty.
```

Comment:
The grammar here seems off.

I would either change it to "sends notifications" or remove "can".

Reply:
Yep, my mistake, fixing it!